chcemvediet/templates/styleguide/components/main.html

```diff
@@ -21,4 +21,7 @@
     <div class="col-sm-4">{% include "styleguide/components/carousel.html" %}</div>
     <div class="col-sm-4">{% include "styleguide/components/sidebar.html" %}</div>
   </div>
+  <div class="row">
+    <div class="col-sm-6 col-md-8">{% include "styleguide/components/casestudy.html" %}</div>
```

Comment:
@martinmacko47 V style guide mam pouzit rovnake `col-sm` a `col-md` classy ako je na stranke?

Reply:
`class="col-sm-6 col-md-8"` je rozumna volba. Nech sa to sprava ako v realite.